### PR TITLE
Remove changecase plugin. Functionality is provided by GtkSourceView.

### PR DESCRIPTION
**Pull request by mbkma**
Add shortcuts to docbook. Remove Change Case Plugin.
New shortcuts:
Screenshot at 2020-07-26 10-13-50

### CODE REVIEW

File: help/C/index.docbook [hunk 1]
```diff
@@ -2109,30 +2047,6 @@
...
-                        </listitem>
-                        <listitem>
-                            <para>To remove the indentation, choose <menuchoice> <guimenu>Edit</guimenu> <guimenuitem>Unindent</guimenuitem> </menuchoice>.</para>
-                        </listitem>
-                    </itemizedlist>
-                </listitem>
-            </orderedlist>
-            <para>The amount of space used, and whether tab character or space characters are used, depends on the <guilabel>Tab Stops</guilabel> settings in the Editor Preferences: see <xref linkend="pluma-prefs-editor"/>.</para>
-        </section>
-
```
Comment (rbuj) [inline help/C/index.docbook:2135]: This edit should be done in a separate commit, since it's not related with the removal of Change Case plugin.

Comment (mbkma) [reply help/C/index.docbook]: Yes, you are right.

**Comment by sc0w** [comment]
I notice the same shortcuts in gedit, but I don't have the <kbd>~</kbd> key (spanish keyboard), and I need to press <kbd>control</kbd><kbd>Alt Gr</kbd><kbd>4</kbd>, maybe is better a combination which works in the same way in all keyboards?

**Comment by mbkma** [comment]
@rbuj thanks, I adopted your changes.
@sc0w I changed the shortcuts to
Screenshot at 2020-08-02 22-27-49

**Comment by sc0w** [comment]
@mbkma  35 commits behind `mate-desktop:master` , can you update it please?

**Comment by mbkma** [comment]
@sc0w done

**Comment by sc0w** [comment]
```
pluma-ui.h:106:2: warning: missing initializer for field 'accelerator' of 'GtkActionEntry' {aka 'const struct _GtkActionEntry'} [-Wmissing-field-initializers]
  106 |  { "ChangeCase", NULL, N_("C_hange Case") },
      |  ^
```

**Comment by mbkma** [comment]
This is a warning that appears for all other toplevel GtkActionEntries as well, maybe its better to fix in another PR?
I think just adding `NULL,NULL,NULL`, i.e. `{ "ChangeCase", NULL, N_("C_hange Case"), NULL, NULL, NULL}` would do it.

**Comment by rbuj** [comment]
@mbkma Can you do it in another PR including the other cases? #568 (If so, I will remove the PR)

**Comment by mbkma** [comment]
I just applied `{ "ChangeCase", NULL, N_("C_hange Case"), NULL, NULL, NULL}`
@rbuj I think #568 is fine

**Comment by rbuj** [comment]
I think there's nothing remaining to be done, @sc0w do you agree?

File: pluma/pluma-commands.h [hunk 2]
```diff
@@ -103,6 +103,14 @@ void		_pluma_cmd_edit_paste			(GtkAction   *action,
...
 void		_pluma_cmd_edit_delete			(GtkAction   *action,
 							 PlumaWindow *window);
+void            _pluma_cmd_edit_upper_case              (GtkAction   *action,
+                                                         PlumaWindow *window);
+void            _pluma_cmd_edit_lower_case              (GtkAction   *action,
+                                                         PlumaWindow *window);
+void            _pluma_cmd_edit_invert_case             (GtkAction   *action,
+                                                         PlumaWindow *window);
+void            _pluma_cmd_edit_title_case              (GtkAction   *action,
+                                                         PlumaWindow *window);
```
Comment (sc0w) [inline pluma/pluma-commands.h:113]: please respect old style with same indentation

File: pluma/pluma-view.h [hunk 3]
```diff
@@ -95,6 +95,12 @@ void		 pluma_view_cut_clipboard 	(PlumaView       *view);
 void		 pluma_view_copy_clipboard 	(PlumaView       *view);
 void		 pluma_view_paste_clipboard	(PlumaView       *view);
 void		 pluma_view_delete_selection	(PlumaView       *view);
+
+void             pluma_view_upper_case_selection    (PlumaView	 *view);
+void             pluma_view_lower_case_selection    (PlumaView	 *view);
+void             pluma_view_invert_case_selection   (PlumaView	 *view);
+void             pluma_view_title_case_selection    (PlumaView	 *view);
```
Comment (sc0w) [inline pluma/pluma-view.h:102]: please respect old style with same indentation
